src/composables/multisigAccountCreate.ts

```diff
@@ -88,54 +88,83 @@ export function useMultisigAccountCreate({ store }: IDefaultComposableOptions) {
-      innerTx: true,
-      onAccount: gaAccount,
-    });
+  /**
+   * First step of creating the multisig account
+   * Prepare multisig account creation transaction
+   */
+  async function prepareVaultCreationRawTx(payerId: string, accountId: string) {
+    if (!pendingMultisigCreationTxs.value[accountId]?.signedAttachTx) {
+      throw Error(`GA Attach Tx not found for account ${accountId}, Prepare attach transaction first`);
+    }
 
-    // Wrap signed GA attach transaction
-    const payedTx = await sdk.payForTransaction(signedAttachTx, {
-      waitMined: true,
-      modal: false,
-      innerTx: true,
-    });
+    const sdk = await getSdk();
 
-    rawTx = payedTx.rawTx;
-    accountId = gaAccount.publicKey;
+    // Wrap signed GA attach transaction
+    const opt = {
+      ...sdk.Ae.defaults,
+      ...{
+        waitMined: true,
+        modal: false,
+        innerTx: true,
+        fromAccount: payerId,
+      },
+    };
+    const payedTx = await sdk.send(
+      await sdk.payingForTx({
+        ...opt,
+        payerId,
+        tx: pendingMultisigCreationTxs.value[accountId].signedAttachTx,
+      }),
+      opt,
+    );
+    pendingMultisigCreationTxs.value[accountId].rawTx = payedTx.rawTx;
 
     // Calculate fee
-    const { tx } = TxBuilder.unpackTx(rawTx);
+    const { tx } = TxBuilder.unpackTx(payedTx.rawTx);
     const outerFee = tx.encodedTx.tx.fee;
     const innerFee = tx.encodedTx.tx.tx.tx.encodedTx.tx.fee;
     const creationFeeUnformatted = new BigNumber(outerFee).plus(innerFee).toFixed();
     multisigAccountCreationFee.value = Number(AmountFormatter.toAe(creationFeeUnformatted));
+    multisigAccountCreationPhase.value = MULTISIG_CREATION_PHASES.signed;
   }
 
   /**
    * Second step of account creation.
    * @throws Error
    */
-  async function multisigAccountCreate() {
-    if (multisigAccountCreationPhase.value !== MULTISIG_CREATION_PHASES.prepared) {
-      throw new Error('To create new multisig vault the account should be prepared first.');
+  async function deployMultisigAccount(accountId: string) {
+    if (!pendingMultisigCreationTxs.value[accountId]?.signedAttachTx) {
+      throw Error(`Raw PayForTransaction not found for account ${accountId}, Prepare PayForTransaction first`);
     }
 
     const sdk = await getSdk();
 
     // Send transaction to the chain
     const { txHash } = await sdk.api.postTransaction(
-      { tx: rawTx },
+      { tx: pendingMultisigCreationTxs.value[accountId].rawTx },
     );
-    await sdk.poll(txHash);
-    multisigAccountCreationPhase.value = MULTISIG_CREATION_PHASES.deployed;
-
-    const gaContract = await sdk.getAccount(accountId);
-    multisigAccountCreationPhase.value = MULTISIG_CREATION_PHASES.created;
-    multisigAccount.value = {
-      contractId: gaContract.contractId,
-      multisigAccountId: accountId,
-    };
+    const pollingResponse = await sdk.poll(txHash, { blocks: DEFAULT_WAITING_HEIGHT });
```

Comment:
We should wrap this in try catch, because it returb an error if time out
https://github.com/aeternity/aepp-sdk-js/blob/2b528b75355729441b685a3c13fa00b5bccc9b80/src/chain.ts#L90